.github/workflows/github-actions.yml

```diff
@@ -30,6 +30,8 @@ jobs:
 
       - name: "Install Foundry"
         uses: "foundry-rs/foundry-toolchain@v1"
+        with:
+          version: 1.2.3-stable
```

Comment:
there is a new foundry version causing the  tasks for `test-protocol` and `check address generation` to failed